src/System.IO.FileSystem/src/System/IO/FileInfo.cs

```diff
@@ -296,6 +296,17 @@ public void MoveTo(String destFileName)
             Invalidate();
         }
 
+        public FileInfo Replace(String destinationFileName, String destinationBackupFileName)
+        {
+            return Replace(destinationFileName, destinationBackupFileName, false);
```

Comment:
>false [](start = 75, length = 5)

nit, when passing bools it is often clearer to write with named parameter eg

...destinationBackupName, ignoremetadataErrors: false)